src/main/resources/db/migration/V1__Create_customer_table.sql

```diff
@@ -3,4 +3,31 @@ CREATE TABLE customer
     id         BIGINT(20) NOT NULL AUTO_INCREMENT PRIMARY KEY,
     first_name VARCHAR(90),
     last_name  VARCHAR(90)
-)
+);
+
+CREATE TABLE organization
+(
+    uuid           VARCHAR(50) NOT NULL PRIMARY KEY,
+    school_name    VARCHAR(100) NOT NULL,
+    school_address VARCHAR(200) NOT NULL,
+    verification_status VARCHAR(10) NOT NULL,
+    created_by     VARCHAR(50) NOT NULL,
+    created_date   TIMESTAMP,
+    modified_by    VARCHAR(50) NOT NULL,
+    modified_date  TIMESTAMP
+);
+
+CREATE TABLE files
+(
+    uuid          VARCHAR(50) NOT NULL PRIMARY KEY,
+    org_id        VARCHAR(50) NOT NULL REFERENCES organization,
+    relative_path VARCHAR(200) NOT NULL,
+    file_name     VARCHAR(100) NOT NULL,
```

Comment:
Probably we should use something more generic here, like URI.

Like this way the table is usable for file system and also for s3.

Example of URI for a file:

```
Path p3 = Paths.get(URI.create("file:///Users/joe/FileTest.java"));
```

For a S3 would be something like:

```
s3://<bucket-name>/<key>
```